### PR TITLE
Dashboard-Page

### PR DESCRIPTION
**Pull request by PDhanrajnath**

### CODE REVIEW

File: src/components/Pages/DashboardPage/DashboardPage.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+});
+
+export const DashboardPage: React.FC = () => {
+  const classes = useStyles();
+  return (
+    <div className={classes.root}>
+      <SideBar />
+      <div className={classes.topnav}>
+        <TopNavBar
+          location="East Maredpally, Secunderabad"
```
Comment (rohithZE) [inline src/components/Pages/DashboardPage/DashboardPage.tsx:41]: Any reason for hardcoding this?

File: src/components/Pages/DashboardPage/DashboardPage.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+            />
+            <JobList
+              src={hp}
+              days="2d"
+              jobRole="Software Engineer"
+              companyName="Hp"
+              location="Hitech City, Telangana"
+              commuteRoute={["car", "bike", "metro"]}
+            />
+          </div>
```
Comment (rohithZE) [inline src/components/Pages/DashboardPage/DashboardPage.tsx:71]: Put all these in a list in constants.
map on that constants and render joblist

File: src/components/Pages/DashboardPage/DashboardPage.tsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+          <div className={classes.jobdesc}>
+            <JobDescription
```
Comment (rohithZE) [inline src/components/Pages/DashboardPage/DashboardPage.tsx:73]: any reason for hardcoding all this info?

File: src/components/Pages/DashboardPage/DashboardPage.tsx (hunk 2 above)
Comment (PDhanrajnath) [reply src/components/Pages/DashboardPage/DashboardPage.tsx:71]: ok Rohith

File: src/components/Pages/DashboardPage/DashboardPage.tsx (hunk 3 above)
Comment (PDhanrajnath) [reply src/components/Pages/DashboardPage/DashboardPage.tsx:73]: I should get the data from constants file but as of now I hardcoded it
I'll change it and raise another PR  Rohith

**Pull request closed without merging**